packages/react-core/src/demos/Wizard/WizardDemo.md

```diff
@@ -229,3 +394,207 @@ class FullPageWizard extends React.Component {
+          <NavItem itemId={3} isActive={activeItem === 3}>
+            Network Services
+          </NavItem>
+          <NavItem itemId={4} isActive={activeItem === 4}>
+            Server
+          </NavItem>
+        </NavList>
+      </Nav>
+    );
+    const Header = (
+      <Masthead id="basic">
+        <MastheadToggle>
+          <PageToggleButton variant="plain" aria-label="Global navigation">
+            <BarsIcon />
+          </PageToggleButton>
+        </MastheadToggle>
+        <MastheadMain>
+          <MastheadBrand>
+            <Brand src={imgBrand} alt="Patternfly logo" />
+          </MastheadBrand>
+        </MastheadMain>
+      </Masthead>
+    );
+    const Sidebar = <PageSidebar nav={PageNav} />;
+    const pageId = 'main-content-page-layout-default-nav';
+    const PageSkipToContent = <SkipToContent href={`#${pageId}`}>Skip to content</SkipToContent>;
+    const PageBreadcrumb = (
+      <Breadcrumb>
+        <BreadcrumbItem>Section home</BreadcrumbItem>
+        <BreadcrumbItem to="#">Section title</BreadcrumbItem>
+        <BreadcrumbItem to="#">Section title</BreadcrumbItem>
+        <BreadcrumbItem to="#" isActive>
+          Section landing
+        </BreadcrumbItem>
+      </Breadcrumb>
+    );
+
+    const informationPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Information panel content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const configSubstepAContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Configuration substep A content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const configSubstepBContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Configuration substep B content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const additionalPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Additional panel content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const reviewPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Review panel content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const steps = [
+      { id: 0, name: 'Information', component: <p>Step 1 content</p>, drawerPanelContent: informationPanelContent },
+      {
+        id: 1,
+        name: 'Configuration',
+        steps: [
```

Comment:
it looks like "configuration step" does not have. drawer panel content, but it seems to be rendering substep A panel. content when i look at demo

Reply:
This is the expected behavior - each of the substeps on the Configuration step have `drawerPanelContent` which render substep A panel content and substep B panel content!